modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurator/ClientExtensionProjectConfigurator.java

```diff
@@ -677,7 +652,10 @@ private void _configureClientExtensionTasks(
 		TaskProvider<Zip> buildSiteInitializerZipTaskProvider,
 		TaskProvider<CreateClientExtensionConfigTask>
 			createClientExtensionConfigTaskProvider,
-		TaskProvider<DefaultTask> validateClientExtensionIdsTaskProvider) {
+		TaskProvider<DefaultTask> validateClientExtensionIdsTaskProvider,
```

Comment:
Use interface `Task`

Reply:
In order to change to `Task`, I'll have to change it to `Task` all the way up to the original declaration of the original `TaskProvider` like so:
From:
```java
		TaskProvider<DefaultTask> validateClientExtensionIdsTaskProvider =
			GradleUtil.addTaskProvider(
				project, VALIDATE_CLIENT_EXTENSION_IDS_TASK_NAME,
				DefaultTask.class);

		TaskProvider<DefaultTask> validateClientExtensionTaskProvider =
			GradleUtil.addTaskProvider(
				project, VALIDATE_CLIENT_EXTENSIONS_TASK_NAME,
				DefaultTask.class);
```

To:
```java
		TaskProvider<Task> validateClientExtensionIdsTaskProvider =
			GradleUtil.addTaskProvider(
				project, VALIDATE_CLIENT_EXTENSION_IDS_TASK_NAME,
				Task.class);

		TaskProvider<Task> validateClientExtensionTaskProvider =
			GradleUtil.addTaskProvider(
				project, VALIDATE_CLIENT_EXTENSIONS_TASK_NAME,
				Task.class);
```

Is that what you had in mind?

Reply:
yup